packages/headless-react/typedoc/documents/hooks.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Hooks
+---
+
+# Hooks
+
+This package provides React hooks for controllers defined in the Engine Definition. Hooks allow you to access controller state and methods easily within your React components.
+
+## Usage
+
+1. Configure the Engine
+
+Define the controllers you need in the engine configuration. This example includes `Summary` and `ProductList` controllers:
+
+```
+const config : CommerceEngineDefinitionOptions {
+    configuration: {
+    ...getSampleCommerceEngineConfiguration(),
+  },
+  controllers: {
+    summary: defineSummary(),
+    productList: defineProductList(),
+  }
+}
+```
+
+2. Create the Engine definition
+
+Use `defineCommerceEngine` to initialize the engine with your configuration:
```

Comment:
```suggestion
Use `defineCommerceEngine` to define the engine with your configuration:
```
(as far as I understand, the definition is used both server and client-side, with the "initialization" happening partly server side and partly client side)